data_steward/ci/setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Create buckets
+gsutil mb -c nearline -p ${APPLICATION_ID} gs://${DRC_BUCKET_NAME}
```

Comment:
We'll need a variable for `chs` also now. Using it for the `merge_test` for now.

Reply:
Ok I'll add it in advance.